### DISCUSSION TITLE
git shellexecuteex failed Error

### DISCUSSION THREAD

**Discussion by VARWA** [git-for-windows/git#4397] [Q&A]
I have these problems with git Bash. Help, please
![image](https://user-images.githubusercontent.com/60575285/234653343-38961fdc-24fd-4d68-88ab-93a5b6bec602.png)
![image](https://user-images.githubusercontent.com/60575285/234653683-9c5e067c-b2e8-4b04-a4bd-a5987d60c38a.png)

**Comment by PhilipOakley** [comment]
This looks like a local setup problem. 

The screenshots indicate you are in the `Git 2.40.1 Setup` and you have reach the point where the `ReleaseNotes.html` are to be displayed, and that attempt at display failed (shellexecuteex failed: code 1155).

I guess you pressed OK, and the second screen came up and you then got the `/usr/bin/bash --login -i` window that should be starting up a bash window into which you can type the posix shell style commands to operate Git.

Do the Git icons 'work' (i.e. when clicked, a fresh bash window is started) once you have closed the error message dialog and `exit`ed the bash window?

Competing the info template normally provided for issues does help, along with including any messages as plain text and may don't like too many 'picture' (too easy to hide hacks in them)

**Reply by VARWA** [reply]
I got this message in Bash after opening it:
'C: \WINDOWS\system32\drivers\etc\protocol' -> '/etc/protocols'
/usr/bin/cp: cannot create regular file '/etc/protocols': Permission denied
	0 [bash] bash 1623 dofork: child -l - forked process 16392 died unexpected
ly, retry 0, exit code 0xC0000005, errno 11
							bash: fork: retry: Resource temporari
  

ly unavailable
'C:\WINDOWS\systen32\drivers\etc\services' -> '/etc/services'
/usr/bin/cp: cannot create regular file '/etc/services': Permission denied
1263840 [bash] bash 1623 dofork: child -l - forked process 17248 died unexpected
ly, retry 0, exit code 0xC0000005, errno 11
							bash: fork: retry: Resource temporari


ly unavailable
'C:\WINDOWS\system32\drivers\etc\networks' -> '/etc/networks'
/usr/bin/cp: cannot create regular file '/etc/networks': Permission denied
rm: cannot renove '/etc/post-install/01-devices.post': Permission denied
rm: cannot remove '/etc/post-install/03-mtab.post': Permission denied
rm: cannot remove '/etc/post-install/06-windows-files.post': Permission denied
rm: cannot remove '/etc/post-install/99-post-install-cleanup.post': Permission denied

 
userBPC-1 MINGW64 ~
$ 

After closed the window, to re-open with administrator rights. The same problem appeared, now the application does not close, as before. This problem appeared about two weeks ago. I have reinstalled Git several times. The error (shellexecuteex failed: code 1155) only appeared on the last install.
